ee/frontend/mobile-replay/transform.test.ts

```diff
@@ -59,6 +59,76 @@ describe('replay/transform', () => {
             posthogEEModule = await posthogEE()
         })
 
+        test('can process top level screenshot', () => {
+            expect(
+                posthogEEModule.mobileReplay?.transformToWeb([
+                    {
+                        data: { width: 300, height: 600 },
+                        timestamp: 1,
+                        type: 4,
+                    },
+                    {
+                        windowId: '5173a13e-abac-4def-b227-2f81dc2808b6',
+                        data: {
+                            wireframes: [
+                                {
+                                    base64: 'image-content',
+                                    height: 914,
+                                    id: 151700670,
+                                    style: {
+                                        backgroundColor: '#F3EFF7',
+                                    },
+                                    type: 'screenshpt',
```

Comment:
```suggestion
                                    type: 'screenshot',
```

Reply:
I couldn't find this misspelling elsewhere - wonder how the snapshot is passing 🤔

Reply:
🤣